testing/test_deploy_app.py

```diff
@@ -23,7 +26,10 @@
 IAM_SCOPE = 'https://www.googleapis.com/auth/iam'
 OAUTH_TOKEN_URI = 'https://www.googleapis.com/oauth2/v4/token'
 METHOD = 'GET'
-logging.getLogger('googleapiclient.discovery_cache').setLevel(logging.ERROR)
+SERVICE_HEALTH = Gauge('deployment_service_status',
+                              '0: normal; 1: deployment not successful; 2: service down')
+PROBER_HEALTH = Gauge('prober_health',
```

Comment:
Why gauges and not a counter?
What if we have N instances of the prober deploying in parallel?

If we have a counter for these values e.g. 1 counter for successes and 1 for failure then we can just increment those counters and compute rates in order to setup alerts.

Reply:
Want to use value to indicate error type.
If we have n instance, then
```any(signal != 0)``` should trigger alert.

Reply:
Added SUCCESS_COUNT and FAILURE_COUNT